src/story_protocol_python_sdk/resources/License.py

```diff
@@ -111,23 +220,23 @@ def registerCommercialUsePIL(self, minting_fee: int, currency: str, royalty_poli
         except Exception as e:
             raise e
 
-    def registerCommercialRemixPIL(self, minting_fee: int, currency: str, commercial_rev_share: int, royalty_policy: str, tx_options: dict = None) -> dict:
+    def registerCommercialRemixPIL(self, default_minting_fee: int, currency: str, commercial_rev_share: int, royalty_policy: str, tx_options: dict = None) -> dict:
         """
         Convenient function to register a PIL commercial remix license to the registry.
 
-        :param minting_fee int: The fee to be paid when minting a license.
+        :param default_minting_fee int: The fee to be paid when minting a license.
         :param currency str: The ERC20 token to be used to pay the minting fee.
         :param commercial_rev_share int: Percentage of revenue that must be shared with the licensor.
         :param royalty_policy str: The address of the royalty policy contract.
         :param tx_options dict: [Optional] The transaction options.
         :return dict: A dictionary with the transaction hash and the license terms ID.
         """
         try:
-            complete_license_terms = get_license_term_by_type(PIL_TYPE['COMMERCIAL_REMIX'], {
-                'mintingFee': minting_fee,
+            complete_license_terms = self.license_terms_util.get_license_term_by_type(self.license_terms_util.PIL_TYPE['COMMERCIAL_REMIX'], {
+                'defaultMintingFee': default_minting_fee,
                 'currency': currency,
-                'commercialRevShare': int((commercial_rev_share / 100) * 100000000),
-                'royaltyPolicy': royalty_policy,
+                'commercialRevShare': commercial_rev_share,
```

Comment:
For commercial revenue share, the value 100,000,000 corresponds to 100% of the share. Adding this clarification to the comments could improve clarity for future reference.